e2e-tests/visual-regression/README.md

```diff
@@ -0,0 +1,23 @@
+# Visual regression tests
+
+This test suite uses [cypress-image-snapshot](https://github.com/jaredpalmer/cypress-image-snapshot)
+to compare screenshots of pages or elements with a saved snapshot.
+
+To add a test, add a page to `src/pages`, then add a test to `cypress/integration`, or add to an existing spec.
```

Comment:
Maybe a quick note about running the suite using `yarn test`. It's the same as other test suites, but we have those instructions in most READMEs.